### PR TITLE
The embeddable version of the CZ was implemented

### PR DESCRIPTION
**Pull request by novikov-alexander**
It seems that I have some problems with code formatting again

### CODE REVIEW

File: Source/Chronozoom.UI/Chronozoom.UI.csproj [hunk 1]
```diff
@@ -54,6 +54,10 @@
     <NoWarn>1591</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <Content Include="App_Data\ChronoZoom.mdf" />
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/Chronozoom.UI.csproj]: To remove

**Comment by alterm4nn** [comment]
Why did you called it minified? It's embeddable, AFAIK.

File: Source/Chronozoom.UI/css/cz.css [hunk 2]
```diff
@@ -256,6 +254,15 @@ iframe[sutra=picker] + div {
   color: #fff;
 }
 /* level 1 */
+#miniMnu {
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/css/cz.css]: wat

File: Source/Chronozoom.UI/czmin.html [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+                    AddScript("/ui/overlay.js");
+                    AddScript("/scripts/czmin.js");
+                }
+            }
+
+        });
+    </script>
+</head>
+<body>
+    <div id="content" class="fleft" style="height: calc(100% - 25px);
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/czmin.html]: inline css? really?

File: Source/Chronozoom.UI/ui/auth-edit-timeline-form.js.orig [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+                                    delete self.timeline.animation;
+
+                                    // Move to new created timeline
+                                    self.timeline.onmouseclick();
+                                });
+                            } else {
+                                // Move to new created timeline
+                                self.timeline.onmouseclick();
+                            }
+<<<<<<< HEAD
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/ui/auth-edit-timeline-form.js.orig]: Unresolved conflict in merge, please fix

File: Source/Chronozoom.UI/ui/auth-edit-timeline-form.min.js.map.orig [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+"version":3,
+"file":"auth-edit-timeline-form.min.js",
+"lineCount":1,
+<<<<<<< HEAD
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/ui/auth-edit-timeline-form.min.js.map.orig]: unresolved conflct

File: Source/Chronozoom.UI/ui/auth-edit-timeline-form.js.orig [hunk 6]
```diff
@@ -0,0 +1,289 @@
+/// <reference path='../ui/controls/formbase.ts'/>
```
Comment (alterm4nn) [inline Source/Chronozoom.UI/ui/auth-edit-timeline-form.js.orig]: orig file -> incomplete merge

File: Source/Chronozoom.UI/css/cz.css (hunk 2 above)
Comment (novikov-alexander) [inline Source/Chronozoom.UI/css/cz.css]: what's wrong?

Comment (novikov-alexander) [inline Source/Chronozoom.UI/css/cz.css]: I need a separate instance of the menu to make its "ul"s and "li"s unclickable

File: Source/Chronozoom.UI/ui/auth-edit-timeline-form.min.js.map.orig (hunk 5 above)
Comment (novikov-alexander) [inline Source/Chronozoom.UI/ui/auth-edit-timeline-form.min.js.map.orig]: is this files are necessary for commit?

File: Source/Chronozoom.UI/css/cz.css (hunk 2 above)
Comment (alterm4nn) [inline Source/Chronozoom.UI/css/cz.css]: why 'miniMnu' and not 'miniMenu' then?

Comment (novikov-alexander) [inline Source/Chronozoom.UI/css/cz.css]: recall it to miniMenu?

Comment (alterm4nn) [inline Source/Chronozoom.UI/css/cz.css]: yes, rename

**Pull request merged**